python/cuml/neighbors/kneighbors_classifier_mg.pyx

```diff
@@ -25,6 +25,7 @@ from cuml.neighbors.nearest_neighbors_mg import NearestNeighborsMG
 
 from pylibraft.common.handle cimport handle_t
 from cuml.common.opg_data_utils_mg cimport *
+from cuml.metrics.distance_type cimport DistanceType
```

Comment:
Instead of having this defined in two places, why don't we start using `pylibraft.distance.DistanceType` instead?